<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordSheetMaker</class>
 <widget class="QWidget" name="WordSheetMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>913</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>40</y>
     <width>261</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	border: 2px solid rgb(0, 0, 0);
	border-radius: 10px;
}</string>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QCheckBox" name="check_numbering">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Numbering</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_shuffle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Shuffle</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_answer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>정답 파일 제작</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_foreign">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>외국어</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_korean">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>한국어</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="drag_and_drop">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>741</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 4px dashed rgb(170, 170, 170);
	border-radius: 5px;
	background-color: rgb(231, 231, 231);
}

QPushButton:hover{
	border: 4px dashed rgb(170, 170, 170);
	border-radius: 5px;
	background-color: rgb(200, 200, 200);
}</string>
   </property>
   <property name="text">
    <string>Drag And Drop</string>
   </property>
  </widget>
  <widget class="QListWidget" name="file_list">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>741</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 4px solid rgb(0, 0, 0);
border-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="make">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>440</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(0, 0, 0);
	border-radius: 10px;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover{
	border: 2px solid rgb(0, 0, 0);
	border-radius: 10px;
	background-color: rgb(230, 230, 230);
}</string>
   </property>
   <property name="text">
    <string>Make</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="output">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>330</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	border: 2px solid rgb(0, 0, 0);
	border-radius: 10px;
}</string>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QRadioButton" name="text">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
